src/Idris/Syntax.idr

```diff
@@ -596,10 +598,19 @@ mutual
     showPrec d (PDotted _ p) = "." ++ showPrec d p
     showPrec _ (PImplicit _) = "_"
     showPrec _ (PInfer _) = "?"
-    showPrec d (POp _ op x y) = showPrec d x ++ " " ++ showPrec d op ++ " " ++ showPrec d y
+    showPrec d (POp _ op x y) =
```

Comment:
Same remark here: please lift `showPrecOp : Prec -> Name -> String` out and refactor
each use site as `showPrec d x ++ " " ++ showPrecOp d op ++ " " ++ showPrec d y`

Reply:
fixed. this patch leads to some tests failed for the expected `x::xs` turns out to be `x :: xs` or `xs++ys` to `xs ++ ys`. I think these failures are kind of ... hmm ... work as expected. should I accept those new values?

Reply:
well, I had done diff in the wrong direction, nevermind